CMakeLists.txt

```diff
@@ -282,22 +290,39 @@ if(WITH_SPDK)
   set(HAVE_SPDK TRUE)
 endif(WITH_SPDK)
 
-option(WITH_PMEM "Enable PMEM" OFF)
-if(WITH_PMEM)
-  set(HAVE_PMEM ON)
-  if(NOT WITH_BLUESTORE)
-    message(SEND_ERROR "Please enable WITH_BLUESTORE for using PMEM")
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "i386|i686|amd64|x86_64")
+  # NVML is now PMDK
+  if(WITH_PMEM_DEVICE)
+    option(WITH_PMDK "Enable PMDK libraries" ON)
```

Comment:
might want to use `CMAKE_DEPENDENT_OPTION`. see https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html .

Reply:
Done